RecoTauTag/RecoTau/interface/DeepTauBase.h

```diff
@@ -140,614 +141,1080 @@ namespace deep_tau {
   };
 
   namespace Scaling {
-    enum class FeatureT{TauFlat, GridGlobal, PfCand_electron, PfCand_muon, PfCand_chHad, PfCand_nHad, PfCand_gamma, Electron, Muon};
-    struct ScalingParams
-    {
-        const std::vector<std::vector<float>> mean_;
-        const std::vector<std::vector<float>> std_;
-        const std::vector<std::vector<float>> lim_min_;
-        const std::vector<std::vector<float>> lim_max_;
-    };
 
-    const std::map<FeatureT, ScalingParams> scalingParamsMap_v2p1 = {
-        {FeatureT::TauFlat, {
+    const std::map<std::pair<FeatureT, bool>, ScalingParams> scalingParamsMap_v2p1 = {
```

Comment:
sorry, my comment was about moving everything related to scaling, including constants into `DeepTauScaling.h` in order to minimize changes in `DeepTauBase.h`